pavelib/assets.py

```diff
@@ -240,17 +292,16 @@ def compile_sass(options):
         output_style = 'compressed'
 
     timing_info = []
-    system_sass_directories = applicable_sass_directories(systems)
-    all_sass_directories = applicable_sass_directories()
+    system_sass_directories = sass_source_directories(systems)
+    all_sass_directories = sass_lookup_directories()
```

Comment:
I think this should still be scoped to current system: `sass_lookup_directories(system)`. I am not sure why it was always adding all paths (lms + cms) to the load path before, but that shouldn't be necessary. Any file that is shared between lms and cms sass should go into the common sass directory; we shouldn't be adding all directories to the load path here.